source/documentation/deploying_services/postgresql.md

```diff
@@ -271,6 +271,115 @@ cf update-service SERVICE_NAME -c '{"reboot": true, "force_failover": true}'
 
 When you force a failover, your PostgreSQL database IP address will change. The database's hostname will not change. You must configure your app to close all database connections to the previous IP address after forcing a failover.
 
+### Add or remove extensions for a Postgresql service instance
+
+The following extensions are always enabled for Postgresql service instances:
+
+<div style="height:1px;font-size:1px;">&nbsp;</div>
+
+|Extension|PostgreSQL version|
+|:---|:---|
+|uuid-ossp|9.5, 10|
+|postgis|9.5, 10|
+|citext|10|
+
+<div style="height:1px;font-size:1px;">&nbsp;</div>
+
+You cannot add or remove these mandatory extensions from a Postgresql service instance.
+
+GOV.UK PaaS supports the following optional extensions, but they are not enabled by default:
+
+<div style="height:1px;font-size:1px;">&nbsp;</div>
+
+|Extension|PostgreSQL version|
+|:---|:---|
+|pg_stat_statements|9.5, 10|
+
+<div style="height:1px;font-size:1px;">&nbsp;</div>
+
+You can enable or disable optional extensions for a PostgreSQL service instance by:
+
+- creating a new service instance
+- updating an existing service instance
+- restoring a PostgreSQL service instance snapshot
+
+#### Create a new service instance
+
+You [create a new PostgreSQL service instance](#set-up-a-postgresql-service) by running `cf create-service`. You can enable additional optional extensions in this new service instance by running:
```

Comment:
`additional optional` - imo this is somewhat cluttering, I think just saying `additional` reads better

Reply:
will just use optional to link it to the first section's split between mandatory and optional